### DISCUSSION TITLE
Context Deadline Exceeded Errors

### DISCUSSION THREAD

**Discussion by VipraB** [aquasecurity/trivy#4970] [Q&A]
### Question

We are facing and error of context deadline exceeded even after the scan is completed 100%

100.00% 0 p/s2023-08-09T17:00:27.328+0530	[31mFATAL[0m	k8s scan error: scanning misconfigurations error: scan error: scan failed: failed analysis: post analysis error: post analysis error: helm scan error: scan config error: context deadline exceeded

### Target

Kubernetes

### Scanner

Vulnerability

### Output Format

Template

### Mode

Client/Server

### Operating System

CentOS Linux release 7.9.2009 (Core)

### Version

```bash
trivy version
Version: 0.44.0
```

**Comment by knqyf263** [comment]
For example, doesn't `--timeout 60m` help?
